scripts/sell_unused_slots.py

```diff
@@ -41,6 +45,46 @@
 log = logging.getLogger(__name__)
 
 
+@attr.s(auto_attribs=True, kw_only=True, frozen=True)
+class Lambda:
+    name: str
+    slot_location: str
+
+    @property
+    def is_contribution_lambda(self) -> bool:
+        for lambda_name in self._contribution_lambda_names():
+            try:
+                # FIXME: Eliminate hardcoded separator
+                #        https://github.com/databiosphere/azul/issues/2964
```

Comment:
Are this new additions? should they be added as an independent commit like 'Add FIXME (#…)'?

Reply:
No, this FIXME was already present, it just moved when I moved this method.
https://github.com/DataBiosphere/azul/blob/8b0cabcc3f8bb0f97fb55aa4633276b5ffc21b28/scripts/sell_unused_slots.py#L99